### DISCUSSION TITLE
Thoughts about Simple Search

### DISCUSSION THREAD

**Discussion by pranjaldugar** [pocketbase/pocketbase#2199] [Show and tell]
In relation Picker UI,
It would be nice to be able to search `john` instead of `name ~ "john"`.
It would make things easier for less technical admins and also be faster to type.

Here `name` can be referenced from `display fileds` choosen.

**Implementation**
In frontend, when search string is pressed, we can first check for operators (=,>,~,! Etc) if none of them are present we can proceed to send `{display field} ?~ {string} to backend`.

Thoughts? :)

**Comment by ganigeorgiev** [comment]
I've added it to the roadmap to consider it.

It could be implemented but for consistency the same search behavior would have to be added also for the regular records table listing and I'm not entirely sure how well it will work when the search term is applied on multiple fields (maybe each field joined with `||`?). I'll have to test it locally first.
